docs/modules/operation/pages/deep-dive/admin/configuration/secure-cookie.adoc

```diff
@@ -0,0 +1,27 @@
+
+= Secure Cookie Attribute
+:description: Learn how to configure the Secure Cookie attribute in {page-component-title} to protect the session cookie confidentiality.
+
+The Secure Cookie attribute controls whether browsers should enforce the encrypted transmission of cookies.
+This prevents some types of attacks, as the enforced encrypted transmission prevents the session cookie from being read and modified.
+
+{page-component-title} ships with a default HTTP configuration and therefore the Secure Cookie Attribute for the session cookie is set to `false`.
+For productive environments in which HTTPS is used, it is highly recommended to activate this flag.
```

Comment:
Should be `production` instead of `productive`.

Reply:
done.